From 5872aa76fb0f767def5e079e0c67307b74df7154 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Thu, 29 Aug 2019 00:49:41 +0300
Subject: [PATCH] configs: rcar-gen3-common: Enable loadb commands

This adds CONFIG_CMD_LOADB to rcar-gen3-common.h
which enables load{b|s|x|y} commands. These commands
allow image download via serial interface.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/rcar-gen3-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/rcar-gen3-common.h b/include/configs/rcar-gen3-common.h
index 3dbfa0a..c574ac3 100644
--- a/include/configs/rcar-gen3-common.h
+++ b/include/configs/rcar-gen3-common.h
@@ -33,6 +33,7 @@
 #define CONFIG_CMD_EXT4_WRITE
 #define CONFIG_CMD_CACHE
 #define CONFIG_CMD_GPIO
+#define CONFIG_CMD_LOADB
 
 #define CONFIG_CMD_SOURCE
 #define CONFIG_CMD_ECHO
-- 
2.7.4

